pipelines/upload/schema/dial/Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+@Library('deploy-conf') _
+node() {
+    try {
+        String ANSI_GREEN = "\u001B[32m"
+        String ANSI_NORMAL = "\u001B[0m"
+        String ANSI_BOLD = "\u001B[1m"
+        String ANSI_RED = "\u001B[31m"
+        String ANSI_YELLOW = "\u001B[33m"
+
+        stage('checkout public repo') {
+            folder = new File("$WORKSPACE/.git")
+            if (folder.exists())
+            {
+               println "Found .git folder. Clearing it.."
+               sh'git clean -fxd'
+            }
+            checkout scm
+        }
+
+            ansiColor('xterm') {
+                values = [:]
+                currentWs = sh(returnStdout: true, script: 'pwd').trim()
+                envDir = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-3].trim()
+                module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
+                jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
+                stage('deploy artifact'){
+                    sh """
+                rm -rf sunbird-dial-service
+                git clone https://github.com/project-sunbird/sunbird-dial-service.git -b ${params.dial_branch_or_tag}
+                    """
+                    ansiblePlaybook = "${currentWs}/ansible/dial_upload-schema.yml"
+                    ansibleExtraArgs = "--extra-vars \" source_name=${currentWs}/sunbird-dial-service/schemas \" --vault-password-file /var/lib/jenkins/secrets/vault-pass"  
+                    values.put('currentWs', currentWs)
+                    values.put('env', envDir)
+                    values.put('module', module)
+                    values.put('jobName', jobName)
+                    values.put('ansiblePlaybook', ansiblePlaybook)
+                    values.put('ansibleExtraArgs', ansibleExtraArgs)
+                    println values
+                    ansible_playbook_run(values)
+                    currentBuild.result = 'SUCCESS'
```

Comment:
Add the option to display values of job parameters used in the job history panel.
![Screenshot from 2022-03-11 11-58-32](https://user-images.githubusercontent.com/3581357/157814527-42045be0-2000-4583-ab2f-53f3550b12ef.png)